Library/Homebrew/extend/ENV/super.rb

```diff
@@ -23,7 +23,8 @@ def self.extended(base)
   end
 
   def self.bin
-    (HOMEBREW_REPOSITORY/"Library/ENV").subdirs.reject { |d| d.basename.to_s > MacOS::Xcode.version }.max
+    bin = (HOMEBREW_REPOSITORY/"Library/ENV").subdirs.reject { |d| d.basename.to_s > MacOS::Xcode.version }.max
+    bin.realpath unless bin.nil?
```

Comment:
Can you change this to `(HOMEBREW_LIBRARY/"ENV").realpath.subdirs.reject { ...`

That should have the same effect, but avoids the nil check.